script.js

```diff
@@ -1,4 +1,74 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+  var spsOutput = ['scissors', 'paper', 'stone']
+  var setSpsOutput =  Math.floor(Math.random()*spsOutput.length)
+  var comChoice = spsOutput[setSpsOutput]
+  console.log (comChoice)
+
+  if (input == comChoice) {
+    var myOutputValue = "It's a draw."
+  } 
+
+  else if (input == 'reversed scissors' && comChoice == 'scissors' || input == 'reversed paper' && comChoice == 'paper'|| input == 'reversed stone' && comChoice == 'stone') {
+    var myOutputValue = "It's a draw."
+  }
+
+  else if (input == 'scissors') {
+    if (comChoice == 'paper') {
+      var myOutputValue = "You won!"
+    }
+    else if (comChoice == 'stone') {
+      var myOutputValue = "You lost!"
+    }
+  }   
```

Comment:
There is something know as [SCREAM_CASE constants ](https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/)

I can touch more on this as the days go by, but it's simply another way to name variables that remain unchanged through out the program eg.
```
var STONE = "stone"
var SCISSORS = "scissors"
var PAPER = "paper"
```